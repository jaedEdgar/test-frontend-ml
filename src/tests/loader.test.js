import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Loader from "../client/components/loader"
import '@testing-library/jest-dom'

test('should render my component', () => {
  const component = renderer.create(<Loader />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
