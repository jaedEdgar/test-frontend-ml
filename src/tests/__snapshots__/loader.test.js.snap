// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render my component 1`] = `
<div
  className="loader-wrapp"
>
  <div
    className="loader"
  >
    <div />
    <div />
  </div>
</div>
`;
